build_tools/scripts/cmake_build.sh

```diff
@@ -26,6 +26,6 @@ CMAKE_BIN=${CMAKE_BIN:-$(which cmake)}
 cd ${ROOT_DIR?}
 rm -rf build/
 mkdir build && cd build
-"$CMAKE_BIN" -DIREE_BUILD_COMPILER=ON -DIREE_BUILD_TESTS=ON -DIREE_BUILD_SAMPLES=OFF -DIREE_BUILD_DEBUGGER=OFF ..
+"$CMAKE_BIN" -DCMAKE_BUILD_TYPE=Debug -DIREE_BUILD_COMPILER=ON -DIREE_BUILD_TESTS=ON -DIREE_BUILD_SAMPLES=OFF -DIREE_BUILD_DEBUGGER=OFF ..
```

Comment:
I was saying I think this should be FastBuild

```suggestion
"$CMAKE_BIN" -DCMAKE_BUILD_TYPE=FastBuild -DIREE_BUILD_COMPILER=ON -DIREE_BUILD_TESTS=ON -DIREE_BUILD_SAMPLES=OFF -DIREE_BUILD_DEBUGGER=OFF ..
```

Reply:
You're absolutely right. Sorry for that, as an excuse, it's quite late here.

Reply:
Oh, I could have picked that directly here on GitHub. Wasn't aware of this feature. Next time :)